packer/openstack.pkr.hcl

```diff
@@ -150,19 +150,20 @@ variable "groups" {
     # fat image builds:
     openhpc = ["control", "compute", "login"]
     openhpc-ofed = ["control", "compute", "login", "ofed"]
+    openhpc-cuda = ["control", "compute", "login", "ofed", "cuda"]
   }
 }
 
 source "openstack" "openhpc" {
   # Build VM:
   flavor = var.flavor
-  use_blockstorage_volume = var.use_blockstorage_volume
+  use_blockstorage_volume = true
```

Comment:
Seem reasonable but worth noting that we require cinder to build locally.

Reply:
Doesn't everywhere have this? If you don't use this then your image size = build vm root disk size, which doesn't give you much flexibility. Although I guess that might be OK ...

(to be clear I'm happy to add this back in if this is a problem, it doesn't require a rebuild!)